catalyst/core/callbacks/criterion.py

```diff
@@ -4,14 +4,14 @@
 import torch
 
 from catalyst import utils
-from catalyst.dl.core import Callback, CallbackOrder, RunnerState
+from catalyst.core import _State, Callback, CallbackOrder
```

Comment:
adding `_` to the beginning usually means private classes or private methods in Python.

use just `State`